packages/web-client/app/components/action-card-container/section/index.hbs

```diff
@@ -0,0 +1,9 @@
+<section class="action-card-section" ...attributes>
+  {{#if @title}}
+    <header class="action-card-section__header">
+      {{@title}}
+    </header>
```

Comment:
I feel like this section should have logo and icon arguments that are laid out nicely with the header, this feels more comfortable than absolutely positioning logos to align with this header.

![image](https://user-images.githubusercontent.com/39579264/119097520-6aab7d00-ba47-11eb-901e-2c3aff2212fa.png)

![image](https://user-images.githubusercontent.com/39579264/119097699-a3e3ed00-ba47-11eb-8bbe-768ec83f334a.png)

Reply:
@Aierie I made the header change to include icons and added the missing icon. Made the logo a background-image since it only serves a presentation purpose as you've said and we don't need to dynamically manipulate its colors etc like we do with svg icons. I made the ActionCardContainer::Title as a separate component from ActionCardContainer::Section, so Title can be used separately if needed with custom content, but just passing in the same arguments to Section also work. I named it "Title" and not "Header" because we already have a Boxel::Header component.